lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -14,6 +14,9 @@ def self.github_client
         user = client.user
         UI.message("Logged in as: #{user.name}")
 
+        # Auto-paginate to ensure we're not missing data
+        client.auto_paginate = true
```

Comment:
We don't want to silently miss anything if there are more than 100 PRs, so I've turned this on